portfolio/src/main/webapp/chart.html

```diff
@@ -5,13 +5,19 @@
     <link rel="stylesheet" href="style.css">
     <!--Load the AJAX API-->
     <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
-    <script src = "script.js" ></script>
+    <script src = "c.js" ></script>
```

Comment:
Is this a second javascript file?

Reply:
Yes that's the javascript file for the chart, I will add a comment to clarify but for some reason the name chart.js would not work.

Reply:
Ah okay, that's weird but hopefully it doesn't give you any rendering issues :)